File: python/MongoDB.py
# importing module
from pymongo import MongoClient

hostname = "<$hostname>"
database = "<$database>"
port = "<$port>"
username = "<$user>"
password = "<$password>"

uri = "mongodb://" + username + ":" + password + "@" + hostname + ":" + port + "/" + database
# creation of MongoClient
client = MongoClient()

# Connect with the portnumber and host
client = MongoClient(uri)

# Access database
mydatabase = client[database]

File: c#/MongoDB.cs
using System;
using MongoDB.Driver;

public class MongoDB
{
    public static void Main()
    {

        uri = "mongodb://<$user>:<$password>@<$hostname>:<$port>/<$database>";
        MongoClient dbClient = new MongoClient(uri);

        // ping
        var ping = dbClient.GetDatabase("admin").RunCommandAsync((Command<BsonDocument>)"{ping:1}").Result;
        Console.WriteLine(ping.ToJson());

        Console.WriteLine("Done.");
        dbClient = null;
    }
}
